libretro_core_options.h

```diff
@@ -834,7 +834,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(pgxp_vertex),
       "PGXP Vertex Cache",
       NULL,
-      "Allows PGXP-enhanced vertex positions to be cached for re-use across polygon draws. Can potentially improve object alignment and reduce visible seams when rendering textures, but false positives when querying the cache may produce graphical glitches. It is currently recommended to leave this option disabled. This option is applied only when PGXP Operation Mode is enabled. Only supported by the hardware renderers.",
+      "Cache PGXP-enhanced vertex positions for re-use across polygon draws. Can potentially improve object alignment and reduce visible seams when rendering textures, but false positives when querying the cache may produce graphical glitches. It is currently recommended to leave this option disabled. This option is applied only when PGXP Operation Mode is enabled. Only supported by the hardware renderers.",
```

Comment:
"by the hardware renderers" needs the article?

Reply:
Probably not. But it's also a good distinction "supported by the hardware renderers we provide, but not by the software renderers". It works either way. Might as well keep it as it is, right?

Reply:
Okay then, to be ignored then.